README.md

```diff
@@ -1 +1,107 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+**[Portfolio]** is create a Portfolio and include my apps on it
+
+## 🛠 Built With <a name="Majd Alkilany"></a>
+
+### Tech Stack <a name="HTML andCSS"></a>
+
+HTML and CSS
+
+### Key Features <a name="create nice and will disign portfolio "></a>
+
+- **[Portfolio ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Code Editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`git clone git@github.com:majdalkilany/portfolio-.git`
+
+### Install
+
+No need To install any dependency
+
+### Usage
+
+run the server from the live-server extinction
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+to get started with this project just visit this [link](https://majdalkilany.github.io/portfolio-s/)
+
+## 👥 Authors <a name="authors"></a>
+
+Majd Alkilany
+
+👤 **Author1**
+
+- GitHub: [@majdalkilany](https://github.com/majdalkilany)
+- Twitter: [@majdkilany92](https://twitter.com/majdkilany92)
+- LinkedIn: [MajdAlkilany](https://www.linkedin.com/in/majdalkilany/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **add som interactivity to the website**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+thanks for visit my first website in Microverse journey
+If you like this project please support me in this
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank every one how incurge me to start this journey
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
[OPTIONAL]
- [ ] please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1: